### PR TITLE
ChatView: don't timestamp old messages separator in append_log_lines()

### PR DESCRIPTION
**Pull request by slook**
Minor UI change:

+ Changed: Don't timestamp the `--- old messages above ---` separator line, because the complexity required in the code it is not really worthwhile for this purpose, actually it is visually easier to spot it with no timestamp.

Minor code cleanups:

+ Changed: the lowercase `login_username` string is required for each line in lines, so it needs doing only once beforehand
+ Changed: Call the `read_log()` method directly from gui module instead of parsing config values into the widget
+ Removed: no caller gets the value that was returned by append_line()

This structure of methods is part of what will be required for future enhancements that are planned in draft PR #2792 (please try running that branch and let me know what you think of the overall idea), but this PR is ready to be merged, if desired.

### CODE REVIEW

File: pynicotine/gtkgui/chatrooms.py [hunk 1]
```diff
@@ -658,14 +658,14 @@ def read_room_logs_finished(self):
...
-            folder_path=log.room_folder_path,
-            basename=self.room,
-            num_lines=config.sections["logging"]["readroomlines"],
-            timestamp_format=config.sections["logging"]["rooms_timestamp"]
-        )
+        log_lines = self._read_old_messages()
+
+        self.chat_view.append_log_lines(log_lines, login_username=config.sections["server"]["login"])
+
+    def _read_old_messages(self):
```
Comment (mathiascode) [inline pynicotine/gtkgui/chatrooms.py:667]: Would probably make more sense to get rid of this function for now and just call log.read_log() directly?

File: pynicotine/gtkgui/chatrooms.py [hunk 2]
```diff
@@ -658,14 +658,14 @@ def read_room_logs_finished(self):
...
+        return log.read_log(log.room_folder_path, self.room, config.sections["logging"]["readroomlines"])
```
Comment (mathiascode) [inline pynicotine/gtkgui/chatrooms.py:668]: Could you pass the arguments as keyword arguments instead? Would be easier to see what they do at a glance.

File: pynicotine/gtkgui/privatechat.py [hunk 3]
```diff
@@ -394,14 +394,14 @@ def read_private_log_finished(self):
...
-            basename=self.user,
-            num_lines=config.sections["logging"]["readprivatelines"],
-            timestamp_format=config.sections["logging"]["private_timestamp"]
-        )
+        log_lines = self._read_old_messages()
+
+        self.chat_view.append_log_lines(log_lines, login_username=config.sections["server"]["login"])
+
+    def _read_old_messages(self):
+        return log.read_log(log.private_chat_folder_path, self.user, config.sections["logging"]["readprivatelines"])
```
Comment (mathiascode) [inline pynicotine/gtkgui/privatechat.py:404]: See previous comments.

File: pynicotine/gtkgui/widgets/textview.py [hunk 4]
```diff
@@ -49,7 +47,6 @@ class TextView:
 
     except TypeError:
         # Broken cursor theme, but what can we do...
-        log.add_debug("Cannot load cursors from theme, falling back to default cursor")
```
Comment (mathiascode) [inline pynicotine/gtkgui/widgets/textview.py:52]: You suggested this log message originally.

File: pynicotine/gtkgui/widgets/textview.py [hunk 5]
```diff
@@ -405,24 +400,19 @@ def __init__(self, *args, chat_entry=None, status_users=None, username_event=Non
...
-
-        lines = log.read_log(folder_path, basename, num_lines)
+    def append_log_lines(self, log_lines, login_username=None):
 
-        if not lines:
+        if not log_lines:
             return
 
-        login = config.sections["server"]["login"]
+        login_lower = login_username.lower() if login_username else None
```
Comment (mathiascode) [inline pynicotine/gtkgui/widgets/textview.py:408]: login_username_lower

File: pynicotine/gtkgui/chatrooms.py (hunk 1 above)
Comment (slook) [reply pynicotine/gtkgui/chatrooms.py]: Yes, it's not required until textview is made capable of prepending lines as in PR #2792. In there, I found that a seg fault may occur when rapidly infinite scrolling (such as when holding down the PageUp key) unless GLib.idle_add() was used, which doesn't accept multiple arguments hence why this separate function call was implemented.

File: pynicotine/gtkgui/widgets/textview.py (hunk 4 above)
Comment (slook) [reply pynicotine/gtkgui/widgets/textview.py:52]: We have finished debugging that issue now, so it is no longer required (and so, the log import is not needed anymore for this is the only call to it).

Iirc our conclusion was that some cursor themes use different naming schemes, and indeed there is nothing we can do about that.

**Pull request merged**